packages/dashboard/src/utils/utils.ts

```diff
@@ -125,5 +125,5 @@ export const reverseLookup = async (
 };
 
 export const shortenAddress = (address: string) => {
-  return `${address.substr(0, 6)}...${address.substr(address.length - 4, 4)}`;
```

Comment:
Looks like you will need to adjust the 2nd argument for the `slice` method here. In `substr` the 2nd argument is the length of the string you want to extract. In slice it is the index to extract up to.